src/sniffer/cmd/main.go

```diff
@@ -33,9 +33,7 @@ func main() {
 		TimestampFormat: time.RFC3339,
 	})
 	errorreporter.Init("sniffer", version.Version(), viper.GetString(sharedconfig.TelemetryErrorsAPIKeyKey))
-	if viper.GetBool(sharedconfig.DebugKey) {
```

Comment:
Why deleting the debug setting?

Reply:
it's a duplicate, appears twice in this file